tools/df/docker/bird-fi-test/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# base image 
+ARG BASE_IMAGE=276327358223.dkr.ecr.us-east-1.amazonaws.com/pipedream-dev
+ARG TAG=latest
+ARG BIRD_TAG=2.0.4-4.df
+ARG ARCH=amd64
+
+FROM ${BASE_IMAGE}:${TAG}
+
+ENV BIRD_TAG=${BIRD_TAG}
+ENV ARCH=${ARCH}
+
+WORKDIR /home/support
+ADD media media
```

Comment:
What is media?

Also I'm not sure you need pipedream-dev as the starting docker image if you're just running bird; Ubuntu should suffice.